backend/hitas/views/apartment_list.py

```diff
@@ -34,13 +36,17 @@ class Meta:
 
 class ApartmentListSerializer(EnumSupportSerializerMixin, HitasModelSerializer):
     state = HitasEnumField(enum=ApartmentState)
-    type = serializers.CharField(source="apartment_type.value")
+    type = serializers.SerializerMethodField()
     address = ApartmentHitasAddressSerializer(source="*")
     surface_area = HitasDecimalField()
     completion_date = serializers.DateField(required=False, allow_null=True)
     ownerships = OwnershipSerializer(many=True, read_only=False)
     links = serializers.SerializerMethodField()
 
+    @staticmethod
+    def get_type(instance: Apartment) -> Optional[str]:
+        return getattr(getattr(instance, "apartment_type", None), "value", None)
+
```

Comment:
Would be a little cleaner if this was squashed with the same fix on the previous commit :)